tests/integration/seed/attendancesSeed.js

```diff
@@ -298,97 +298,103 @@ const courseHistoryList = [
     company: authCompany._id,
     createdBy: userList[2]._id,
     trainee: traineeList[0]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[0]._id,
     company: authCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[3]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[0]._id,
     company: authCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[8]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[1]._id,
     company: authCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[0]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[2]._id,
     company: otherCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[9]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[2]._id,
     company: otherCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[2]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
   },
   {
     action: TRAINEE_ADDITION,
     course: coursesList[3]._id,
     company: otherCompany._id,
     createdBy: userList[0]._id,
     trainee: traineeList[2]._id,
-  },
-  {
-    action: TRAINEE_ADDITION,
-    course: coursesList[3]._id,
-    company: authCompany._id,
-    createdBy: userList[0]._id,
-    trainee: traineeList[2]._id,
+    createdAt: '2023-01-03T14:00:00.000Z',
```

Comment:
pourquoi cet ajout ?

Reply:
sans les dates de création, le test qui vérifie qu'on est dans la bonne structure quand on est rattaché à la formation casse (should pass if all trainees are in course company at registration)